### ISSUE TITLE
rake task error

### ISSUE THREAD

**Issue by ksenia-zalesnaya** [TalentBox/sequel_bitemporal#4]
When I tried to deploy new code with migration I got this error:
```
rake aborted!
Sequel::DatabaseError: PG::UndefinedTable: ERROR:  relation "player_versions" does not exist
LINE 1: SELECT * FROM "player_versions" LIMIT 1
                      ^
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:186:in `async_exec'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:186:in `block in execute_query'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/database/logging.rb:44:in `log_connection_yield'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:186:in `execute_query'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:173:in `block in execute'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:149:in `check_disconnect_errors'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:173:in `execute'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:525:in `_execute'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:341:in `block (2 levels) in execute'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:546:in `check_database_errors'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:341:in `block in execute'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/database/connecting.rb:251:in `block in synchronize'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/connection_pool/threaded.rb:107:in `hold'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/database/connecting.rb:251:in `synchronize'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:341:in `execute'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/dataset/actions.rb:970:in `execute'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/adapters/postgres.rb:669:in `fetch_rows'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/dataset/actions.rb:141:in `each'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/dataset/actions.rb:73:in `columns'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/model/base.rb:248:in `columns'
/.../.rvm/gems/ruby-2.3.1/gems/sequel_bitemporal-0.8.5/lib/sequel/plugins/bitemporal.rb:45:in `configure'
/.../.rvm/gems/ruby-2.3.1/gems/sequel-4.36.0/lib/sequel/model/base.rb:681:in `plugin'
/.../app/models/player.rb:2:in `<class:Player>'
/.../app/models/player.rb:1:in `<top (required)>'
```
That was because of `plugin :bitemporal, version_class: PlayerVersion` in model. In development I didn't notice that since I added this code after migration, but for production I had to deploy 2 times - migration first, then the code.

**Comment by JonathanTron** [comment]
@ksenia-zalesnaya Are you using the model in your migration?

**Comment by fiscal-cliff** [comment]
Hello, @ksenia-zalesnaya, @JonathanTron 
I guess, the problem is that the plugin tries to load all the columns during initialization process. And if you tries to invoke db:drop and then db:create you will run into this problem.
There is quite popular scenario, if you reload all the migrations on CI server and versions table won't exist during the migration process. That's why the build will be marked as failed.

**Comment by fiscal-cliff** [comment]
Consider adding check for versions table
https://github.com/TalentBox/sequel_bitemporal/pull/5

**Comment by JosephHalter** [comment]
Fixed in version 0.8.7
